.travis.yml

```diff
@@ -45,12 +45,10 @@ before_install:
     - export PATH=/usr/lib/jvm/java-8-oracle/jre/bin/:$PATH
     - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
     - java -version
-    - ant -version
     - mysql --version
     - sqlite3 --version
     - env
     - cd wrappers/java
-    - ant clean jar
```

Comment:
Can you make maven compile the source code here (i.e. the equivalent of `ant jar`) ? I'm sure calling `mvn test` will trigger a compilation automatically, like `ant run_tests` would, but again this is to make sure everything is ok early